### ISSUE TITLE
Demon running but no sound 

### ISSUE THREAD

**Issue by jamie-marchant** [lukasjapan/bt-speaker#26]
Hi: 
 The demon is running but I get no sound, my dmesg is filled with "Bluetooth: hci0: Frame reassembly failed".

**Comment by romankc** [comment]
I was able to resolve  "Bluetooth: hci0: Frame reassembly failed" by booting into CLI mode (instead of desktop). Were you ever able to get sound to come out?

**Comment by jamie-marchant** [comment]
No, let me charge my Android(which I use for testing) and see if it works.

**Comment by jamie-marchant** [comment]
Did not work for me :(.

**Comment by romankc** [comment]
I have the same problem. I opened a new issue for it. I will let you know if I manage to find/create a fix.
